bin/dev/create_helm_charts_pr.sh

```diff
@@ -42,6 +42,10 @@ unzip -d bundle/ bundle.zip
 bundle=$(basename "$CAP_BUNDLE" .zip)
 bundle_readable=$(printf '%b' "${bundle//%/\\x}")
 product=`echo $bundle_readable | cut -d+ -f1`
+
+# Strip -rcN from the product version shown in PR description.
+product="$(echo "${product}" | sed -e 's|-rc.*||')"
```

Comment:
```suggestion
product="$(echo "${product}" | sed -e 's|-rc.*$||')"
```
Not sure that's actually needed here.

Reply:
Hm. The `-rc...` is always at the end, AFAIK/CT. Maybe go with digits as well, instead of the general `.*`.

Reply:
Actually, thinking about it ... Using `-rc[0-9][0-9]*` generally should simplify replacements (always empty), and will find everything, regardless of location. Trialing that now.

Reply:
Addressed in commit a38786f3eb1a4343fe7bb3a205775774729fa753.